federated_search/auth-service/tests/test_auth.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import os
+import sys
+from fastapi.testclient import TestClient
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
I'm kind of surprised this is needed... is this so that you can run test_auth.py as a script, instead of running it with the CLI...?